core/src/Cabana_DeepCopy.hpp

```diff
@@ -385,6 +404,175 @@ inline void deep_copy( Slice_t &slice,
 
 //---------------------------------------------------------------------------//
 
+/**
+ * @brief Copy a partial chunk from src into the destination array
+ *
+ * @param dst
+ * @param src
+ * @param start_from
+ * @param end_from
+ * @param start_to
+ */
+template <class DstAoSoA, class SrcAoSoA>
+inline void deep_copy_partial_src(
+    DstAoSoA &dst, const SrcAoSoA &src,
+    // const int to_index,
+    // TODO: the order of these params is questionable
+    const int from_index, const int count,
```

Comment:
I think `count`, then `index` makes the most sense given both partial deep copy options

Reply:
I still can't decide, I think no I removed the "other" index in both cases, I'll leave it as `{start index, count}` for now